### ISSUE TITLE
Use card-mod-view-yaml to get rid of the need for template: edge

### ISSUE THREAD

**Issue by aFFekopp** [UI-Lovelace-Minimalist/UI#43]
Since you are already using card-mod. You can add padding around the whole page with `card-mod-view-yaml`

```
  card-mod-theme: minimalist-mobile
  card-mod-root: |
    app-toolbar {
      display: none;
    }
  card-mod-view-yaml: |
    "*:first-child$": |
      #columns .column > * {
        padding-left: 5px;
        padding-right: 5px;
        padding-bottom: 5px;
      }   
```

That way you don't need to add

```
- type: custom:button-card
  template: edge
```

Around every card

**Comment by damianeickhoff** [comment]
I think the card could be usefull in some cases where you want to center a card, but not the rest. Still, I think the theme looks overall better with the padding left and right.

**Comment by CM000n** [comment]
Thank you for pointing this out. I will include it in the next release: https://github.com/UI-Lovelace-Minimalist/UI/pull/92

**Comment by CM000n** [comment]
Closed in v1.0.0-beta.8 🏳️‍🌈
